ibis/compat.py

```diff
@@ -1,29 +1,4 @@
 """Module for compatible functions."""
-import operator
 import sys
 
-import toolz
```

Comment:
hmm can we completely remove this module?

Reply:
Yes I agree, `PY38` should probably be moved somewhere else.

At the moment I think it is only used for BigQuery. For example in the udf tests:
https://github.com/ibis-project/ibis/blob/e91e635e852308ed9b0d819a83becf983f5e8cbb/ibis/backends/bigquery/tests/udf/test_core.py#L11
https://github.com/ibis-project/ibis/blob/e91e635e852308ed9b0d819a83becf983f5e8cbb/ibis/backends/bigquery/tests/udf/test_find.py#L10
https://github.com/ibis-project/ibis/blob/e91e635e852308ed9b0d819a83becf983f5e8cbb/ibis/backends/bigquery/tests/udf/test_udf_execute.py#L16

And it is also imported in the following `__init__.py` , but (as far as I can tell) only used in a doctstring:
https://github.com/ibis-project/ibis/blob/e91e635e852308ed9b0d819a83becf983f5e8cbb/ibis/backends/bigquery/udf/__init__.py#L9

Where do you think I should move it to?